draft-tulshibagwale-saag-pushpull-delivery.md

```diff
@@ -74,19 +74,42 @@ normative:
 
 
 --- abstract
-In situations where a transmitter of Security Event Tokens (SETs) to a network peer is also a receiver of SETs from the same peer, it is helpful to have an efficient way of sending and receiving SETs in one HTTP transaction. In many cases, such as when using the OpenID Shared Signals Framework (SSF), the situation where each entity is both a transmitter and receiver is getting increasingly common.
+This specification defines how multiple Security Event Tokens (SETs) can be delivered and received to and by an intended recipeint using HTTP POST over TLS.
```

Comment:
Not true. We are also describing Websockets transport in this spec.

Reply:
@tulshi are you suggesting to update as follows?

> ... _by an intended recipeint using HTTP POST over TLS and/or Websocktes._

Reply:
yes, "WebSockets"